### PR TITLE
STAND-83 make standalone compatible with java 1.8

### PR DESCRIPTION
**Pull request by jdegraft**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1,625 +1,657 @@
-<project xmlns="http//maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```
Comment (wluyima) [inline pom.xml]: James, looks like you have git end of line issues, can you please fix them so that the diff here only shows your changes?

Comment (wluyima) [inline pom.xml]: I had actually addressed this at the summit in Singapore, see https://github.com/openmrs/openmrs-standalone/pull/26

**Comment by jdegraft** [comment]
@wluyima can you review my changes?
your change for tomcat version #26 seems more correct since I was not able to change my proposed change.
I had to change some plugin versions to support java 1.8, including specifying maven-jar-plugin with version 26 since the default it was using without explicit specification was version 24 which did not work with java 1.8

File: pom.xml [hunk 2]
```diff
@@ -12,7 +12,21 @@
...
+		 <!-- tomcat version -->
+		<openmrs.version>2.0.0-SNAPSHOT</openmrs.version>
+		<!-- use -D to specify the os_arch to select mysqld -->
+		<!-- 
+		<os.name>Win</os.name>
+		<os.arch>x86</os.arch>
+		 -->
+		<!-- os_arch -->
+		<javaVersion>1.8</javaVersion>
 	</properties>
```
Comment (wluyima) [inline pom.xml]: Are these changes really necessary? I'm able to build the standalone with java 8 even for platform 2.0 after my pull request without these changes

Comment (jdegraft) [inline pom.xml]: the only change necessary is to use javaVersion 1.8 instead of the currently specified 1.6. If this change is made then maybe it maybe necessary to change the jar plugins.
If (and I clearly don't understand this very well) it is not necessary to change the compiler version in the pom, and your pull request works then this pull request can be closed without merging.
I would appreciate if you explain in a comment why we compile with java 1.8 but leave the compiler spec in the pom as java 1.6. Thanks.

Comment (jdegraft) [inline pom.xml]: @wluyima I think I have it figured out. The java version in the pom is the base compatibility level of the build right, so I was wrong to update it to 1.8? If this is right then I will close this pull request.

Comment (wluyima) [inline pom.xml]: Right now we don't use any new APIs in  java 8, so it fine to leave the target and source versions of the compiler at 1.6, i think the goal is to make it work for platform 2.0 and java 8
